pkg/da/data_availability_header.go

```diff
@@ -182,6 +185,10 @@ func MinDataAvailabilityHeader() DataAvailabilityHeader {
 }
 
 // MinShares returns one tail-padded share.
-func MinShares() [][]byte {
-	return shares.ToBytes(shares.TailPaddingShares(appconsts.MinShareCount))
+func MinShares() ([][]byte, error) {
```

Comment:
[nice to have, but optional]
we could add some documentation here to when an error is thrown to avoid having to dig through the tailpadding shares function.

Reply:
Sure will add it

Reply:
done

Reply:
[no change needed] on re-review it seems strange to return an error from this function because there are no function parameters. I expect `MinShares()` to never return an error. It could `panic()` if `shares.TailPaddingShares(appconsts.MinShareCount)` returns an error but I don't think such a change is blocking.